config/routes.rb

```diff
@@ -111,7 +111,7 @@
     post :comment
     post :update_setting
 
-    post "/disconnect/facebook" => 'users#disconnect_facebook',
+    get "/disconnect/facebook" => 'users#disconnect_facebook',
```

Comment:
We should probably change this on the other side instead. GET should be idempotent